rosdep/base.yaml

```diff
@@ -2310,6 +2310,7 @@ libglew-dev:
   ubuntu:
     artful: [libglew-dev]
     bionic: [libglew-dev]
+    focal: [libglew-dev]
     oneiric: [libglew1.5-dev]
     precise: [libglew1.6-dev]
     quantal: [libglew-dev]
```

Comment:
I know you are just adding a new key here, but I think we can simplify this given the current supported Ubuntu releases.  That is, I think we can remove this whole block and replace it with:

```
ubuntu: [libglew-dev]
```

Which should work on Xenial and newer, and also make us future-proof.